systemrequirements.adoc

```diff
@@ -258,6 +277,11 @@ including:
 
 * Various device/operating system combinations with mobile WAP browser.
```

Comment:
Seems like this whole non-smartphone section should go.  AFAIK, we don't support non-smartphones at this point. WAP in general is dead and I don't recall Zimbra supporting it anyway.  Also that link (non-https by the way) redirects to a generic page about ZCS... nothing mobile specific.